pkg/evaluation/evaluation.go

```diff
@@ -136,9 +136,13 @@ func (eval *RagasDatasetGenerator) Generate(ctx context.Context, csvData io.Read
 
 	// convert langchain documents to ragas dataset
 	for _, doc := range langchainDocuments {
+		groundTruths, ok := doc.Metadata[eval.options.groundTruthsColumn].(string)
+		if !ok {
```

Comment:
should we add some long to print out doc.Metadata[eval.options.groundTruthsColumn] for troubleshooting?